contentcuration/contentcuration/assets/channelList/utils.js

```diff
@@ -0,0 +1,39 @@
+import omit from 'lodash/omit';
+import cloneDeep from 'lodash/cloneDeep';
+import isEqual from 'lodash/isEqual';
+import store from './store';
+
+const DIRTY_KEY = '_lastSaved';
+
+export function setDirtyTracking(object) {
+  // Store a record of previous attributes here for dirty checking.
+  object[DIRTY_KEY] = cloneDeep(omit(object, DIRTY_KEY));
```

Comment:
Storing the copy of the object as a key on itself seems to introduce some extra complexity, and might be risky if `object` is reactive?

I wonder if there's another place these snapshots could be stored. A really simple strategy would be to just make a parallel  `[object]Prev` variation of any objects (`channelList`, `channelListPrev`).

Reply:
Yeah - I do a cloneDeep to prevent the reactive concern, but this the last saved State is also reactive now and doesn't need to be, so would be fine to just store it on somewhere else.